frontend/packages/dev-console/src/components/import/import-validation-utils.ts

```diff
@@ -94,6 +95,84 @@ export const validationSchema = yup.object().shape({
       .string()
       .matches(pathRegex, { message: 'Path must start with /.', excludeEmptyString: true }),
   }),
+  limits: yup.object().shape({
+    cpu: yup.object().shape({
+      request: yup
+        .number()
+        .nullable()
+        .min(0, 'Request must be greater than or equal to 0.')
```

Comment:
Request of 0 doesn't really make sense, although I don't know what the API allows. It's probably OK because I'm not sure what else we'd use a minimum. Maybe greater than 0.